source/Applications/Basic/Visualization/CaptureHDRVisNormals/CaptureHDRVisNormals.cpp

```diff
@@ -0,0 +1,123 @@
+/*
+This example shows how to capture Zivid point clouds, with color and normals.
+For scenes with high dynamic range we combine multiple acquisitions to get an HDR point cloud.
+*/
+
+#include <Zivid/Zivid.h>
+
+#include <pcl/visualization/cloud_viewer.h>
+
+#include <iostream>
+#include <thread>
+
+using namespace std::chrono_literals;
+
+namespace
+{
+    pcl::visualization::PCLVisualizer::Ptr normalsVisualizer(pcl::PointCloud<pcl::PointXYZRGB>::ConstPtr cloud,
+                                                             pcl::PointCloud<pcl::PointXYZRGBNormal>::ConstPtr normals)
+    {
+        // --------------------------------------------------------
+        // -----Open 3D viewer and add point cloud and normals-----
+        // --------------------------------------------------------
+        auto viewer = boost::make_shared<pcl::visualization::PCLVisualizer>("Viewer");
+
+        int view_rgb(0);
+        viewer->createViewPort(0.0, 0.0, 0.5, 1.0, view_rgb);
+        viewer->addText("Cloud RGB", 0, 0, "RGBText", view_rgb);
+        viewer->addPointCloud<pcl::PointXYZRGB>(cloud, "cloud", view_rgb);
+
+        int view_normals(0);
+        viewer->createViewPort(0.5, 0.0, 1.0, 1.0, view_normals);
+        viewer->addText("Cloud Normals", 0, 0, "NormalsText", view_normals);
+        viewer->addPointCloud<pcl::PointXYZRGBNormal>(normals, "cloudNormals", view_normals);
+        viewer->addPointCloudNormals<pcl::PointXYZRGBNormal, pcl::PointXYZRGBNormal>(normals,
+                                                                                     normals,
+                                                                                     10,
+                                                                                     1,
+                                                                                     "normals",
+                                                                                     view_normals);
+
```

Comment:
Please add:
```cpp
std::cout << "Note! Only every 10th normal is visualized." << aperture << std::endl;
```
Maybe even make a variable that is resused in the comment. This way it's even easier for a customer to understand which parameter to modify to control the number of normals to visualize.
```cpp
        int normals_skipped(10);
        viewer->addPointCloudNormals<pcl::PointXYZRGBNormal, pcl::PointXYZRGBNormal>(normals,
                                                                                     normals,
                                                                                     normals_skipped,
                                                                                     1,
                                                                                     "normals",
                                                                                     view_normals);
        std::cout << "Note! Only every " << normals_skipped << "th normal is visualized." << aperture << std::endl;
```

Reply:
done in 709761ea98cc6c5aaa5fce7b8b81353173e3ee6f